openwisp_monitoring/device/api/views.py

```diff
@@ -251,5 +311,18 @@ def _create_clients_graph(self, metric):
         graph.full_clean()
         graph.save()
 
+    def _create_resources_graph(self, metric, resource):
+        if resource not in monitoring_settings.AUTO_GRAPHS:
+            return
+        graph = Graph(metric=metric, configuration=resource)
+        graph.full_clean()
+        graph.save()
+
+    def _create_resources_threshold(self, metric, resource):
+        value = DEVICE_RESOURCES_THRESHOLDS[resource]
+        t = Threshold(metric=metric, operator='>', value=value, seconds=0)
```

Comment:
ok, I think we need a way to specify the default value passed to `seconds`.

I realize now that `seconds` is not a good name because it may not convey what it does, that field indicates the sensitivity of the threshold and is used to avoid constantly sending alerts.

While I was writing this I thought that rather than spending time on this, we'd better investigate in a general solution which I tried to describe in #106.

So nothing to do here now, I just added this comment to explain how and why #106 appeared.